CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Unreleased
```

Comment:
Make this 2.1.1 – also update pubspec.yaml!

Reply:
Done.